packages/manager/src/managers/prismicRepository/PrismicRepositoryManager.ts

```diff
@@ -673,6 +673,29 @@ export class PrismicRepositoryManager extends BaseManager {
 		}
 	}
 
+	async setDefaultMasterLocale(): Promise<void> {
+		const repositoryName = await this.project.getRepositoryName();
+
+		const localeServiceBaseUrl = new URL("locale/", API_ENDPOINTS.PrismicWroom);
+		localeServiceBaseUrl.hostname = `${repositoryName}.${localeServiceBaseUrl.hostname}`;
+		const url = new URL("repository/locales", localeServiceBaseUrl);
+		url.searchParams.set("repository", repositoryName);
+
+		const res = await this._fetch({
+			url,
+			method: "POST",
+			body: {
+				id: "en-us",
+				isMaster: true,
+				label: "English - United States",
```

Comment:
❓ I don't find where did you see a `label` as body param of this call?

Reply:
Yes, you're right, it's not needed.
It's removed.